nodewatcher/modules/frontend/ip_space/templates/network/ip_space/ip_space.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% contextblock %}
+    {% load i18n sekizai_tags html_tags url_tags partial_tags leaflet_tags %}
+
+    {% add_data "css_data" "css/ipspace.css" %}
+    {% add_data "js_data" "js/d3.v4.js" %}
+    {% add_data "js_data" "js/point.js" %}
+    {% add_data "js_data" "js/hilbert2d.js" %}
+    {% add_data "js_data_end" "js/jquery.tipsy.js" %}
+    {% add_data "js_data_end" "js/draw_cidrs.js" %}
+    {% add_data "js_data_end" "js/main.js" %}
+
+    {% trans "Ip space" as title %}
+{% endcontextblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-2">
+            <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Zoom to subnet" %}<span class="caret"></span></button>
+                <ul class="dropdown-menu" id="topnodes">
+                    <li id="cidr"><a href="#" value="10.0.0.0/8">{% trans "Loading data" %}</option>
+                </ul>
+            </div>
+        </div>
+        <div class="col-md-8" id="wraper">
+            <svg id="ipspace" class="silky-smooth"></svg>
```

Comment:
What is this class name? What does it mean?

Reply:
It is a reference to hit movie You Don't Mess with the Zohan from 2008, it does all kinds of stuff for the main map window, do you want me to rename it to map or something like that?

Reply:
Very interesting. Yes, please make it something bring but related to the project.